app/models/employment/BenefitsViewModel.scala

```diff
@@ -106,11 +107,13 @@ case class BenefitsViewModel(
   val assetsDetailsPopulated: Boolean =
     assets.isDefined || assetTransfer.isDefined
 
-  val benefitsPopulated: Boolean = vehicleDetailsPopulated || accommodationDetailsPopulated || travelDetailsPopulated ||
-    utilitiesDetailsPopulated || medicalDetailsPopulated || incomeTaxDetailsPopulated || reimbursedDetailsPopulated || assetsDetailsPopulated
 }
 
 object BenefitsViewModel {
+
+  def clear(isUsingCustomerData:Boolean):BenefitsViewModel =
+    BenefitsViewModel(isBenefitsReceived = false, isUsingCustomerData = isUsingCustomerData)
```

Comment:
I only left this in for readability reasons, so that if you come to look what the clear is doing it shows that the only 2 fields it leaves in are the isBenefitsReceived as false and isUsingCustomerData value. Happy to take it out though if not needed?